crates/agglayer-config/src/storage.rs

```diff
@@ -104,6 +118,36 @@ impl From<StorageConfig> for StorageConfigHelper {
     }
 }
 
+fn default_db_path() -> PathBuf {
+    PathBuf::new().join(STORAGE_DIR)
+}
+
+pub(crate) fn normalize_path(path: &Path) -> PathBuf {
+    let mut components = path.components().peekable();
+    let mut ret = if let Some(c @ Component::Prefix(..)) = components.peek().cloned() {
+        components.next();
```

Comment:
Why is the prefix component removed? A comment explaining it would be useful.

Reply:
This part of the method inherited from `cargo`